browse-scaler/README.md

```diff
@@ -4,7 +4,7 @@ Nodejs application to return thumbnails for NASA's Earthdata Search
 
 # Building
 
-**NOTE**: because Sharp uses C++ extensions, it must be built in a docker container to run on AWS (unless your machine is also linux).
+**NOTE**: because Sharp uses C++ extensions, it must be built in a docker container to run on AWS (unless your machine is also linux). If you are on a mac machine you may need to un-install sharp using: `npm uninstall sharp` and then re-install it using: `npm install --platform=linux --arch=x64 sharp`
```

Comment:
```suggestion
**NOTE**: because Sharp uses C++ extensions, it must be built in a Docker container to run on AWS (unless your machine is also Linux). If you are using macOS you may need to un-install sharp using: `npm uninstall sharp` and then re-install it using: `npm install --platform=linux --arch=x64 sharp`
```